mne_bids/config.py

```diff
@@ -131,7 +131,7 @@
     ALLOWED_INPUT_EXTENSIONS +
     ['.json', '.tsv', '.tsv.gz', '.nii', '.nii.gz'] +
     ['.pos', '.eeg', '.vmrk'] +  # extra datatype-specific metadata files.
-    ['.dat', '.EEG']  # extra eeg extensions
+    ['.mrk', '.dat', '.EEG']  # extra eeg extensions
```

Comment:
thanks :-) is it an "extra eeg extension" though? Or should this rather be:

```suggestion
    ['.dat', '.EEG'] +  # extra eeg extensions
    ['.mrk']  # KIT/Yokogawa/Ricoh marker coil
```